### PR TITLE
SECURESIGN-150 | Modify the screen shown on successful authentication with cosign

### PR DESCRIPTION
**Pull request by JasonPowr**
This pr adds the changes necessary to update the landing page when a user authenticates with keycloak.

## Testing 

1. Apply the overlay files 
2. **Make sure you are using a fresh build of cosign created after the overlay files are applied**
3. Stand up a cluster 
4. Using key cloak and the new cosign binary sign and verify an image  
5. You should be brought to the new landing page instead of the old one 

## Before
image

## After
Screenshot from 2023-11-02 15-41-38

### CODE REVIEW

File: cmd/cosign/cli/fulcio/fulcio.go [hunk 1]
```diff
@@ -92,7 +92,13 @@ func GetCert(_ context.Context, sv signature.SignerVerifier, idToken, flow, oidc
 	case flowDevice:
 		c.flow = oauthflow.NewDeviceFlowTokenGetterForIssuer(oidcIssuer)
 	case flowNormal:
-		c.flow = oauthflow.DefaultIDTokenGetter
+		if strings.Contains(oidcIssuer, "keycloak") {
```
Comment (lance) [inline cmd/cosign/cli/fulcio/fulcio.go:95]: Why this conditional? I think we always want to show branded Red Hat response, no matter what OIDC provider is used.

File: internal/ui/index.go [hunk 2]
```diff
@@ -0,0 +1,226 @@
+package ui
+
+const (
+	KeycloakInteractiveSuccessHTML = `<!DOCTYPE html>
```
Comment (lance) [inline internal/ui/index.go:4]: I think this could be `RedHatInteractiveSuccessHTML` since I don't think it should be limited to keycloak auth only.

**Comment by JasonPowr** [comment]
@lance :)

**Review by Gregory-Pereira — APPROVED** [review]
Hats off Jason, this looks awesome.
/lgtm

**Pull request merged**